dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.tds.ssl/server.xml

```diff
@@ -85,8 +81,8 @@
     
     <sslDefault sslRef="DefaultSSLSettings" />
 	<ssl id="LDAPSSLSettings" keyStoreRef="LDAPKeyStore" trustStoreRef="LDAPTrustStore" />
-	<keyStore id="LDAPKeyStore" location="${server.config.dir}/LdapSSLKeyStore.jks" type="JKS" password="{xor}CDo9Hgw=" />
-	<keyStore id="LDAPTrustStore" location="${server.config.dir}/LdapSSLTrustStore.jks" type="JKS" password="{xor}CDo9Hgw=" />
+	<keyStore id="LDAPKeyStore" location="${server.config.dir}/keystore.p12" type="PKCS12" password="LDAPpassword" />
+	<keyStore id="LDAPTrustStore" location="${server.config.dir}/truststore.p12" type="PKCS12" password="LDAPpassword" />
```

Comment:
I want you to not create the 'truststore.p12' file. This file should copied into this directory in the build.gradle file from the UnboundID project. This is one of the changes I asked for after you finished the PR for the build break. So do it as part of this PR for both this test server and the AD test server.